modules/systemd/systemd.sh

```diff
@@ -3,6 +3,26 @@
 # Tell build process to exit if there are any errors.
 set -euo pipefail
 
+# Copy included systemd unit files to system
+SYSTEM_UNIT_INCLUDE="$CONFIG_DIRECTORY"/systemd/system
+USER_UNIT_INCLUDE="$CONFIG_DIRECTORY"/systemd/user
+SYSTEM_UNIT_DIR="/usr/lib/systemd/system"
+USER_UNIT_DIR="/usr/lib/systemd/user"
+
+shopt -s dotglob
```

Comment:
What does this globbing accomplish?

Reply:
dotglob includes hidden files when you use * expansion.

I enabled this by habit, but I can disable it here for systemd units, as that's not needed.